otbenchmark/ReliabilityProblem60.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue May  5 12:56:49 2020
+
+@author: Jebroun
+
+Class to define the ReliabilityProblem60 benchmark problem.
+"""
+
+from otbenchmark.ReliabilityBenchmarkProblem import ReliabilityBenchmarkProblem
+import openturns as ot
+
+
+class ReliabilityProblem60(ReliabilityBenchmarkProblem):
+    def __init__(
+        self,
+        threshold=0.0,
+        mu1=2200,
+        sigma1=220,
+        mu2=2100,
+        sigma2=210,
+        mu3=2300,
+        sigma3=230,
+        mu4=2000,
+        sigma4=200,
+        mu5=1200,
+        sigma5=480,
+    ):
+        """
+        Creates a reliability problem RP60.
+
+        The event is {g(X) < threshold}
+        We have x1 ~ LogNormal(mu1, sigma1)
+                x2 ~ LogNormal(mu2, sigma2)
+                x3 ~ LogNormal(mu3, sigma3)
+                x4 ~ LogNormal(mu4, sigma4)
+                x5 ~ LogNormal(mu5, sigma5)
+        ***
+        Parameters
+        ----------
+        threshold : float
+            The threshold.
+        mu1 : float
+            The mean of the X1 gaussian distribution.
```

Comment:
In order to clarify this, we should state : "The mean of the underlying gaussian distribution of X1." But this is not consistent with the code, which uses "LogNormalMuSigma", meaning that mu1 is the mean of the LogNormal variable. So there is a bug either in the code or in the doc.

Reply:
I agree with @mbaudin47. To state it differently, from my point of view, for this RP60 benchmark problem, one has:

- (mu1=2200, sigma1=220, ..., mu5=1200, sigma5=480) => They refer to (Mean, Std) in the RP60 table. These values thus correspond to the mean and std of the lognormal random variable. => Use ot.LogNormalMuSigma(mui, sigmai) is fine for me, but the doc is incorrect.

- (theta1=7.691, theta2=0.09975, ...) in the RP60 Table, are the distribution parameters of the underlying Gaussian random variable, and should be set using, e.g., X1 = ot.LogNormal(theta1,theta2).

I don't know what is the simplest bug to fix... (I would say the doc).